README.md

```diff
@@ -1,6 +1,6 @@
 # VLC for iOS & tvOS
 
-This repository has the source code of VLC for iOS and tvOS application.
+This is the official mirror repository of VLC for iOS and tvOS application.
```

Comment:
```diff
-This is the official mirror repository of VLC for iOS and tvOS application.
+This is the official mirror repository of VLC for iOS and tvOS applications.
```
🤓

As we are indicating that this is a mirror. What do you think about linking to the main repo in GitLab here

Reply:
Good idea!